homeassistant/config.py

```diff
@@ -706,11 +706,14 @@ def _recursive_merge(
         if (not hass.config.skip_pip and p_integration.requirements and
                 not await async_process_requirements(
                     hass, p_integration.domain, p_integration.requirements)):
+            _LOGGER.error(
```

Comment:
Isn't `async_process_requirements` already logging if it fails ?

Reply:
Might be so yea. But it won't log correctly in the config check case via the result structure. Nicer to keep them similar (I've a factorisation for this in the works so that share more code, but better to get the bug fixed first)